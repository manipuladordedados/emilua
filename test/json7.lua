local println = require('println')
local json = require('json')

println(tostring(json.decode('30')))
json.decode('30000000000000000000000000000000000000000000000')
