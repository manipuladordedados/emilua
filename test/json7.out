30
Main fiber from VM 0x1 panicked: 'Numerical result out of range'
stack traceback:
	input.lua:5: in main chunk
	[C]: in function ''
	[string "?"]: in function <[string "?"]:0>
